crates/sqllib/src/casts.rs

```diff
@@ -1591,18 +1592,9 @@ where
     vec.into()
 }
 
-pub fn cast_to_V_map<K, V>(map: BTreeMap<K, V>) -> Variant
-where
-    Variant: From<K> + From<V>,
-    K: Clone,
-    V: Clone,
-{
-    map.into()
-}
-
 pub fn cast_to_vec_V<T>(value: Variant) -> Vec<T>
 where
-    Vec<T>: TryFrom<Variant>,
+    Vec<T>: TryFrom<Variant, Error = Box<dyn Error>>,
```

Comment:
Would `<Vec<T>>::Error: Display` be sufficient at this point?

You might have to write it as
```
<Vec<T> as TryFrom<Variant>>::Error: Display,
```